### PR TITLE
send mail to user on rent creation

### PR DESCRIPTION
**Pull request by mtejedorwolox**
**Summary**: Sent mail to user on rent creation.

**Trello Card**: https://trello.com/c/hkrQjLGX

### CODE REVIEW

File: wbooks-api/app/mailers/rent_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class RentMailer < ApplicationMailer
+  default from: 'no-reply@training.wolox.com.ar'
```
Comment (ilanusse) [inline wbooks-api/app/mailers/rent_mailer.rb]: The default `from` should usually be the one in `ApplicationMailer`

File: wbooks-api/app/mailers/rent_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+  def create_notification(rent)
+    @rent = rent
+    mail to: @rent.user.email, subject: 'New Rent'
```
Comment (ilanusse) [inline wbooks-api/app/mailers/rent_mailer.rb]: Keep in mind `subject` should use `I18n` as well.

File: wbooks-api/app/views/rent_mailer/create_notification.html.slim [hunk 3]
```diff
@@ -0,0 +1,46 @@
+h1
+  i
+    | Rent info:
```
Comment (ilanusse) [inline wbooks-api/app/views/rent_mailer/create_notification.html.slim]: I18n for file

File: wbooks-api/app/mailers/rent_mailer.rb (hunk 2 above)
Comment (mtejedorwolox) [inline wbooks-api/app/mailers/rent_mailer.rb]: The i18n it's from another task

File: wbooks-api/app/views/rent_mailer/create_notification.html.slim (hunk 3 above)
Comment (mtejedorwolox) [inline wbooks-api/app/views/rent_mailer/create_notification.html.slim]: same

**Pull request merged**